### PR TITLE
[ZEPPELIN-1162] Fix rawType in NotebookRestApi

### PR DESCRIPTION
**Pull request by rajarajan-g**
### What is this PR for?
Some HashSet is declared without types. Fix it in NotebookRestApi

### What type of PR is it?
Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1162

### How should this be tested?
All existing test cases related to permissions

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by rajarajan-g** [comment]
Please review this PR and let me your thoughts in comments

**Comment by jongyoul** [comment]
I think it's enough. LGTM

File: zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java [hunk 1]
```diff
@@ -116,8 +116,8 @@ public Response putNotePermissions(@PathParam("noteId") String noteId, String re
      * TODO(jl): Fixed the type of HashSet
```
Comment (vectorijk) [inline zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: @rajarajan-g Should we also need to remove comments here?

Comment (jongyoul) [inline zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: Can you handle it? This PR already is closed.

Comment (vectorijk) [inline zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: Sure, I have opened another PR for this.

**Pull request closed without merging**